src/scripts/main.js

```diff
@@ -1,3 +1,18 @@
 'use strict';
 
-// write your code here
+const populationElements = document.querySelectorAll('.population');
+let totalAmount = 0;
+
+populationElements.forEach(popElement => {
+  const populationNum = +popElement.innerText.split(',').join('');
+
+  totalAmount += populationNum;
+});
+
+const averagePopulation = Math.floor(totalAmount / populationElements.length);
+
+const totalPopulationEl = document.querySelector('.total-population');
+const averagePopulationEl = document.querySelector('.average-population');
+
+totalPopulationEl.innerText = totalAmount.toLocaleString();
+averagePopulationEl.innerText = averagePopulation.toLocaleString();
```

Comment:
You should return the number in the format 452,543,964 instead of 452 543 964.
Try to change locale to 'en-US'